homeassistant/components/yeelight/light.py

```diff
@@ -323,52 +315,45 @@ def set_music_mode(self, mode) -> None:
     def update(self) -> None:
         """Update properties from the bulb."""
         import yeelight
-        try:
-            bulb_type = self._bulb.bulb_type
-
-            if bulb_type == yeelight.BulbType.Color:
-                self._supported_features = SUPPORT_YEELIGHT_RGB
-            elif self.light_type == yeelight.enums.LightType.Ambient:
-                self._supported_features = SUPPORT_YEELIGHT_RGB
-            elif bulb_type in (yeelight.BulbType.WhiteTemp,
-                               yeelight.BulbType.WhiteTempMood):
-                if self._is_nightlight_enabled:
-                    self._supported_features = SUPPORT_YEELIGHT
-                else:
-                    self._supported_features = SUPPORT_YEELIGHT_WHITE_TEMP
-
-            if self.min_mireds is None:
-                model_specs = self._bulb.get_model_specs()
-                self._min_mireds = \
-                    kelvin_to_mired(model_specs['color_temp']['max'])
-                self._max_mireds = \
-                    kelvin_to_mired(model_specs['color_temp']['min'])
-
-            if bulb_type == yeelight.BulbType.WhiteTempMood:
-                self._is_on = self._get_property('main_power') == 'on'
-            else:
-                self._is_on = self._get_property('power') == 'on'
-
+        bulb_type = self._bulb.bulb_type
+
+        if bulb_type == yeelight.BulbType.Color:
+            self._supported_features = SUPPORT_YEELIGHT_RGB
+        elif self.light_type == yeelight.enums.LightType.Ambient:
+            self._supported_features = SUPPORT_YEELIGHT_RGB
+        elif bulb_type in (yeelight.BulbType.WhiteTemp,
+                           yeelight.BulbType.WhiteTempMood):
             if self._is_nightlight_enabled:
-                bright = self._get_property('nl_br', None)
+                self._supported_features = SUPPORT_YEELIGHT
             else:
-                bright = self._get_property('bright', None)
+                self._supported_features = SUPPORT_YEELIGHT_WHITE_TEMP
 
-            if bright:
-                self._brightness = round(255 * (int(bright) / 100))
+        if self.min_mireds is None:
+            model_specs = self._bulb.get_model_specs()
+            self._min_mireds = \
+                kelvin_to_mired(model_specs['color_temp']['max'])
+            self._max_mireds = \
+                kelvin_to_mired(model_specs['color_temp']['min'])
 
-            temp_in_k = self._get_property('ct')
+        if bulb_type == yeelight.BulbType.WhiteTempMood:
+            self._is_on = self._get_property('main_power') == 'on'
+        else:
+            self._is_on = self._get_property('power') == 'on'
 
-            if temp_in_k:
-                self._color_temp = kelvin_to_mired(int(temp_in_k))
+        if self._is_nightlight_enabled:
+            bright = self._get_property('nl_br', None)
```

Comment:
The default of _get_property is None by "default". The second parameter isn't needed.